plugins/Goals/API.php

```diff
@@ -151,20 +154,23 @@ private function getModel()
      * @param bool $caseSensitive
      * @param bool|float $revenue
      * @param bool $allowMultipleConversionsPerVisit
+     * @param string $description
      * @return void
      */
-    public function updateGoal($idSite, $idGoal, $name, $matchAttribute, $pattern, $patternType, $caseSensitive = false, $revenue = false, $allowMultipleConversionsPerVisit = false)
+    public function updateGoal($idSite, $idGoal, $name, $matchAttribute, $pattern, $patternType, $caseSensitive = false, $revenue = false, $allowMultipleConversionsPerVisit = false, $description = '')
     {
         Piwik::checkUserHasAdminAccess($idSite);
```

Comment:
I wonder if we should keep the currently saved `$description` if `$description = false` and clear it if `$description = ''`. `false` could be default but I'm not sure what is better and whether is needed / more intuitive. I guess as `caseSensitive` and other optional parameters works the same as implemented it's all good and consistent the way it is implemented now :+1: